src/manage-fonts/font-family.js

```diff
@@ -1,28 +1,36 @@
-import { useState } from 'react';
+import { useContext } from '@wordpress/element';
 import { Button, Icon } from '@wordpress/components';
 import FontFace from "./font-face";
+import { ManageFontsContext } from '../fonts-context';
 
-const { __ } = wp.i18n;
+const { __, _n } = wp.i18n;
 function FontFamily ( { fontFamily, fontFamilyIndex, deleteFontFamily, deleteFontFace } ) {
 
-    const [isOpen, setIsOpen] = useState(true);
+    const { familiesOpen, handleToggleFamily } = useContext(ManageFontsContext);
+    const isOpen = familiesOpen.includes( fontFamily.name || fontFamily.fontFamily );
 
     const toggleIsOpen = () => {
-        setIsOpen(!isOpen);
+        handleToggleFamily( fontFamily.name || fontFamily.fontFamily );
     }
 
-    const hasFontFaces = !!fontFamily.fontFace && !!fontFamily.fontFace.length;
+    const fontFaces = fontFamily.fontFace.filter( face => !face.shouldBeRemoved );
```

Comment:
Hmm, I'm seeing this on the Manage Fonts page with the last commits:

<img width="1403" alt="image" src="https://user-images.githubusercontent.com/1645628/220088820-150ad556-ed4f-4598-92c2-6a24ac3ac61d.png">

I think it's happening when `fontFace` isn't found on line 16.

Reply:
Thanks for testing! it should be solved now